### PR TITLE
kubeless 1.0.0-alpha.1

### PR DESCRIPTION
**Pull request by andresmgot**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by andresmgot** [comment]
@commitay is there any way we can update the recipe until we release 1.0.0?

**Comment by ilovezfs** [comment]
@andresmgot any reason not to just release `1.0.0`? It is very strange to have a current stable spec be designated as an alpha version. Also, do we actually still need to be using the tag not the source tarball if you're no longer setting commit hash in ldflags?

File: Formula/kubeless.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Kubeless < Formula
   desc "Kubernetes Native Serverless Framework"
   homepage "https://github.com/kubeless/kubeless"
   url "https://github.com/kubeless/kubeless.git",
-      :tag => "v0.6.0",
-      :revision => "3bec4b65f29d830bd6709abb07e00f11fe54d646"
+      :tag => "v1.0.0-alpha.1",
```
Comment (ilovezfs) [inline Formula/kubeless.rb]: version is parsed as `1.0.0` (our tag-based, as opposed to url-based, version parsing only looks at digits) so would need an explicit 
```
version "1.0.0-alpha.1"
```

**Comment by andresmgot** [comment]
> @andresmgot any reason not to just release 1.0.0? It is very strange to have a current stable spec be designated as an alpha version.

The main reason is to gather feedback from our users before releasing 1.0.0 (we decided to use `alpha` instead of `rc` for not making a commitment about avoiding major changes before releasing 1.0.0).

> Also, do we actually still need to be using the tag not the source tarball if you're no longer setting commit hash in ldflags?

You are right, it is not needed anymore.

Thanks for your feedback. I'll apply it.

File: Formula/kubeless.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 class Kubeless < Formula
   desc "Kubernetes Native Serverless Framework"
   homepage "https://github.com/kubeless/kubeless"
-  url "https://github.com/kubeless/kubeless.git",
-      :tag => "v1.0.0-alpha.1",
-      :revision => "fac4bb8fe04273e1de161059fecc43a9870aa6a6"
-
+  version "1.0.0-alpha.1"
```
[the remark this answers is not in the document]
Comment (andresmgot) [reply Formula/kubeless.rb]: oh, I see, updating again

**Comment by andresmgot** [comment]
the validation now fails because:

```
kubeless:
  * Stable version URLs should not contain alpha
Error: 1 problem in 1 formula
```

**Comment by ilovezfs** [comment]
we'll ignore it. Thanks @andresmgot. Shipped!

**Pull request closed without merging**